File: app/src/main/java/com/example/theloop/models/CurrentWeather.java
package com.example.theloop.models;

import com.google.gson.annotations.SerializedName;

public class CurrentWeather {

    @SerializedName("time")
    private String time;

    @SerializedName("temperature_2m")
    private double temperature;

    @SerializedName("weather_code")
    private int weatherCode;

    // Getters
    public String getTime() {
        return time;
    }

    public double getTemperature() {
        return temperature;
    }

    public int getWeatherCode() {
        return weatherCode;
    }
}

File: app/src/main/java/com/example/theloop/models/CurrentWeather.kt
package com.example.theloop.models

import com.google.gson.annotations.SerializedName

data class CurrentWeather(
    @SerializedName("temperature_2m") val temperature: Double,
    @SerializedName("weather_code") val weatherCode: Int,
    @SerializedName("time") val time: String
)
